src/webviews/apps/shared/components/onboarding/progress-line.ts

```diff
@@ -0,0 +1,60 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('gl-progress-line')
+export class GlProgressLine extends LitElement {
+	@property({ type: Number })
+	value: number = 0;
+
+	/**
+	 * provides a value that defines the count of possible states of the progress line
+	 */
+	@property({ type: Number })
+	valueRate: number = 20;
+
+	static override readonly styles = css`
+		.progress-container {
+			display: block;
+			height: 7px;
+			width: 100%;
+			border-radius: 4px;
+			background-color: var(--progress-background-color, gray);
```

Comment:
Switch this to `--vscode-progressBar-background`

Reply:
For reference, here's the main color tokens available to use:
- [provided by VSCode](https://code.visualstudio.com/api/references/theme-color)
- [provided by GitLens](https://github.com/gitkraken/vscode-gitlens/blob/16a8590f77f98070f43b05bf677e293f4867e35c/src/webviews/apps/shared/styles/theme.scss)

Reply:
Can I add these links to the confluence (initiatives section)?